src/app.module.ts

```diff
@@ -0,0 +1,42 @@
+import { Module } from '@nestjs/common'
+import { ThrottlerModule, ThrottlerGuard } from '@nestjs/throttler'
+import { APP_GUARD } from '@nestjs/core'
+import { AppController } from './app.controller'
+import { AppService } from './app.service'
+import { ConsoleLogger } from '@credo-ts/core'
+import { IndyVdrPoolConfig } from '@credo-ts/indy-vdr'
+import { Logger } from '@nestjs/common'
+import { IndyVdrProxyModule, setupAgent } from 'credo-ts-indy-vdr-proxy-server'
+import { readFileSync } from 'fs'
+import 'dotenv/config'
+
+const configPath = process.env.INDY_VDR_PROXY_CONFIG_PATH ?? './res/app.config.json'
```

Comment:
Ok for now but we should put this into a config module named config so we can `config.get("blarb")` and supply default values.